src/protocol.rs

```diff
@@ -386,21 +386,29 @@ pub(crate) mod participant_config {
 
 /// An identifier for a [`Participant`].
 ///
-/// All [`Participant`]s in a session must agree on the
+/// All [`Participant`]s in a sub-protocol must agree on the
 /// [`ParticipantIdentifier`]s. That is, these are not local identifiers
 /// controlled by a single `Participant`; they are unique, agreed-upon
-/// identifiers for the `Participant`s in a session. Each entity participating
-/// in a session should have a different `ParticipantIdentifier`.
+/// identifiers for the `Participant`s executing a sub-protocol. Each member
+/// participating in a sub-protocol should have a different `ParticipantIdentifier`.
 ///
-/// `ParticipantIdentifier`s can be used across multiple sessions. For
-/// example, if a set of participants run keygen, auxinfo, and then compute
-/// several signatures, they can use the same set of identifiers for each of
-/// those sessions. However, a single `ParticipantIdentifier` should not be used
-/// to represent different entities (even in different sessions with
+/// The same `ParticipantIdentifier`s are used across multiple sub-protocols. For
+/// example, if a set of participants run keygen, aux-info, and then compute
+/// several signatures, they must use the same set of identifiers for each of
+/// those sessions. These `ParticipantIdentifier`s are embedded in the output
```

Comment:
I'm not sure how I feel about the use of "sessions" in this line after it has been taken out of other spots. It is also currently undefined. I think it would be nice if we could say somewhere that the participant identifiers associated with a single instance of the tss-ecdsa protocol must remain the same throughout all subprotocols (where instance encompasses the lifecycle of a single ECDSA key)

Reply:
Thanks Gabby. I agree with your thoughts. I removed sessions and I added the sentence you wanted. Also tweaked the paragraphs a bit to avoid repetition overall. Lmk what you think.